sample/cf-nodejs-logging-support-sample/public/scripts/logic.js

```diff
@@ -0,0 +1,46 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+    const globalContext = document.getElementById("global-context");
+    const requestContext = document.getElementById("request-context");
+    const customFields = document.getElementById("custom-fields");
+    const stackTrace = document.getElementById("stack-trace");
+    const clientError = document.getElementById("client-error");
+    const childLogger = document.getElementById("child-logger");
+    const correlationTenantId = document.getElementById("correlation-and-tenant-id");
+
+    globalContext.addEventListener("click", () => {
```

Comment:
> Alternatively you could also solve this with a custom attribute assigned to each button. E.g.
> 
> ```
> <button class="btn btn-lg btn-primary btn-block" type="submit" targetpath="globalcontext">...</button>
> ```
> 
> Then assign a common event listener to all buttons based on the assigned class and read the attribute to determine the request path. This would reduce code duplication and make the code more generic.

I applied this solution